com.github.unrud.VideoDownloader.json

```diff
@@ -0,0 +1,116 @@
+{
+    "app-id" : "com.github.unrud.VideoDownloader",
```

Comment:
if this is a new app that you plan to maintain (albeit forked from videodownloader) then you need to change the app-id to match.

Reply:
My original idea was to keep updating video-downloader that is already on Flathub, would that be possible?

Reply:
Except that's not the same app at all but a fork of it. So please use a different app-id in such case.

Reply:
Understandable, will do so